etc/config-template.toml

```diff
@@ -547,10 +547,6 @@
 ## resources of the system and improving TiKV performance in terms of the current resources.
 # compression-per-level = ["no", "no", "lz4", "lz4", "lz4", "zstd", "zstd"]
 
-# Bottommost Compression
-# Sampling dictionary compression may enhance the compression rate when using small data blocks.
-# bottommost_compression: zstd
```

Comment:
We still need this config.

Reply:
Okay, I add a boolean toggle here in the next commit.